app/preprints/-components/submit/preprint-state-machine/status-flow/status-flow-display/component.ts

```diff
@@ -1,18 +1,17 @@
 import Component from '@glimmer/component';
 import PreprintStateMachine from 'ember-osf-web/preprints/-components/submit/preprint-state-machine/component';
-import { inject as service } from '@ember/service';
-import Media from 'ember-responsive';
 
 /**
  * The Status Flow Display Args
  */
 interface StatusFlowDisplayArgs {
     manager: PreprintStateMachine;
+    isMobile: boolean;
```

Comment:
I don't see why we need `isMobile` here as an argument, when we could use the `media` service?